app/screens/main/Main.tsx

```diff
@@ -263,7 +262,7 @@ class Main extends Component<Props, State> {
               </TooltipWrapper>
             </NavBarPart>
           </NavBar>
-          {nodeError ? <InfoBanner /> : <EmptySpace />}
+          <EmptySpace />
```

Comment:
remove it - it's not necessary anymore

Reply:
indeed - bye bye the notorious multi purpose status bar!

Reply:
Done. Removed `EmptySpace` too ;)